### PR TITLE
Remove DEEX and OPEN, remove assets

### PR DESCRIPTION
**Pull request by sschiessl-bcp**
https://dex.openledger.io/news/ol-dex-is-closing-all-activities/

### CODE REVIEW

File: app/branding.js [hunk 1]
```diff
@@ -220,17 +195,6 @@ export function getMyMarketsQuotes() {
...
-            "TWENTIX",
-
-            "CVCOIN",
-            "HERO",
-            "OCT",
-            "HERTZ",
-            "ICOO",
-            "SMOKE",
-            "STEALTH",
-            "YOYOW"
```
Comment (abitmore) [inline app/branding.js:233]: YOYOW is still alive.

File: app/branding.js [hunk 2]
```diff
@@ -272,11 +230,7 @@ export function getFeaturedMarkets(quotes = []) {
...
         ["USD", "RUDEX.SMOKE"],
         ["CNY", "BTS"],
-        ["CNY", "OPEN.BTC"],
         ["CNY", "USD"],
-        ["CNY", "OPEN.ETH"],
-        ["CNY", "YOYOW"],
-        ["CNY", "OCT"],
         ["CNY", "GDEX.BTC"],
         ["CNY", "GDEX.ETH"],
         ["CNY", "GDEX.EOS"],
```
Comment (abitmore) [inline app/branding.js:236]: GDEX announced to delist
* HPB
* BTM
* QTUM
* NEO
* GAS

BTW Rudex delisted a few tokens too, I don't remember if they have created a pull request.

File: app/branding.js [hunk 3]
```diff
@@ -365,7 +298,7 @@ export function getAssetNamespaces() {
     if (_isTestnet()) {
         return [];
     }
-    return ["OPEN.", "RUDEX.", "GDEX.", "XBTSX.", "CITADEL.", "DEEX."];
+    return ["OPEN.", "RUDEX.", "GDEX.", "XBTSX.", "CITADEL."];
```
Comment (abitmore) [inline app/branding.js:301]: Remove OPEN?

File: app/branding.js [hunk 4]
```diff
@@ -220,17 +195,6 @@ export function getMyMarketsQuotes() {
             "XBTSX.EXR"
```
Comment (abitmore) [inline app/branding.js:195]: XBTS added USDT and etc.

File: app/branding.js (hunk 3 above)
Comment (sschiessl-bcp) [reply app/branding.js:301]: Currently marked as disabled with the ! icon.

File: app/branding.js (hunk 4 above)
Comment (sschiessl-bcp) [reply app/branding.js:195]: will ping

**Comment by sschiessl-bcp** [comment]
> Also remove their API nodes?

Never had any in the ref UI

**Pull request merged**